swiftpkg/internal/pkginfo_target_deps.bzl

```diff
@@ -81,15 +81,15 @@ def _resolve_by_name(pkg_ctx, name):
 
     ext_dep = lists.find(
         pkg_ctx.pkg_info.dependencies,
-        lambda d: d.identity == normalized_name,
+        lambda d: d.name == normalized_name,
     )
     if ext_dep != None:
         return [bazel_labels.new(
             name,
             repository_name = bazel_repo_names.from_identity(ext_dep.identity),
             package = "",
         )]
-    fail("Unable to resolve byName reference {name} in {repo_name}.".format(
+    fail("Unable to resolve byName reference {name} in {repo_name}. {deps}".format(
```

Comment:
Do we need to add the `deps` to the format arguments?

Reply:
No, this was debug only, sorry, will remove.